rest-api-spec/src/main/resources/rest-api-spec/api/delete_scroll.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
oh wait though, DELETE is done against _search/scroll while POST is done against _search/clear_scroll . I don't think we can express that in our spec. POST _search/scroll is already taken by the search api.